packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -164,6 +164,12 @@ export function QueueInfoColumn({
           </QueuePropertyText>
         </QueuePropertyRow>
       )}
+
+      {queue?.isProfessorQueue && (
+        <QueuePropertyRow>
+          <Tag>Professor Queue</Tag>
+        </QueuePropertyRow>
+      )}
```

Comment:
Feels like there's a bit too much spacing between the queue title and this tag 🤔 it looks like the `QueueRoomGroup` above it has a sizeable bottom margin. You could get rid of the margin on that and group both the group and the tag together in a div, then set the bottom margin on that div?

Also I just realized there's that thing to display the queue start/end time right above this (line 159) which we don't need anymore - would you mind deleting it while you're here :,,)